extract-text-webpack-plugin/extract-text-webpack-plugin-tests.ts

```diff
@@ -70,10 +70,13 @@ configuration = {
     // ...
     module: {
         rules: [
-            { test: /\.css$/, loader: ExtractTextPlugin.extract({
-                fallbackLoader: "style-loader",
-                loader: "css-loader"
-            }) }
+            {
+                test: /\.css$/,
+                use: ExtractTextPlugin.extract({
+                    fallback: "style-loader",
+                    use: "css-loader"
+                })
+            }
```

Comment:
changing all of this obscured the real change (`loader` → `use`)

i recommend not doing style changes in positions where you also change functionality.